gson/src/main/java/com/google/gson/internal/Streams.java

```diff
@@ -103,6 +105,8 @@ public static Writer writerForAppendable(Appendable appendable) {
     /**
      * A mutable char sequence pointing at a single char[].
      */
+    /*Code is dangerous as functions can be called without initializing chars*/
+    @SuppressWarnings("initialization.fields.uninitialized")
```

Comment:
This class is only used internally, and when it was used `chars` was initialized`